GettingStarted.md

```diff
@@ -1,80 +1,165 @@
+This document describes how to prepare your work environment to use MixedRealityToolkit-Unity in your Unity 3D project.
 
-# 1. Setting up your development environment
+# 1. Check your Windows 10 version
+It is recommended to be running the Windows 10 Fall Creators update for modern Windows Universal apps targeting Mixed Reality Headsets
+
+This can be verified by running the "WinVer" application from the Windows Run command
+
+`Windows Key + R -> WinVer`
+
+Which will display a new window as follows:
+
+![Windows Version dialog](/External/ReadMeImages/WindowsVersionFCU.png)
+
+If you are not running the Windows 10 Fall Creators update, then you will need to Update your version of Windows.
+
+# 2. Setting up your development environment
 Be sure to enable Developer mode for Windows 10 via:
 
 `Action Center -> All Settings -> Update & Security -> For Developers -> Enable Developer mode`
 
 ![Enable Developer Mode](/External/ReadMeImages/EnableDevModeWin10.PNG "Enable Developer Mode for Windows 10")
 
-Be sure to download and install [Visual Studio 2017](https://www.visualstudio.com/vs/) and these required components:
+If you have not already, download and install [Visual Studio 2017](https://www.visualstudio.com/vs/) and these required components:
 
 ![Visual Studio Components](/External/ReadMeImages/VisualStudioComponents.PNG)
 
-If you haven't already installed Unity 3d on your machine, you should [download and install the latest version](https://unity3d.com/get-unity/download/archive) this project says it supports on the [main readme page](/README.md).
+Make sure you are running the appropriate version of Unity 3D on your machine. You should [download and install the latest version](https://unity3d.com/get-unity/download/archive) this project says it supports on the [main readme page](/README.md).
+
+> The Mixed Reality Toolkit now supports the following Unity 3D versions:
+> *  **2017.0.2f3 (for HoloLens)** 
+> * **2017.2.0f3-MTRP (for Immersive headsets)**
 
 _Note: Be sure to include the Windows Store .NET scripting backend components._
 
 ![Unity Installer](/External/ReadMeImages/UnityInstaller.PNG "Unity Installer")
 
-# 2. Download the HoloToolkit-Unity asset packages
-You can download the latest unity package from [Releases](https://github.com/Microsoft/HoloToolkit-Unity/releases) folder.
+# 2. Download the MixedRealityToolkit-Unity asset packages
+You can download the latest unity package from [Releases](https://github.com/Microsoft/MixedRealityToolkit-Unity/releases) folder.
 
-### Using the source code
-Optionally, If you'd like to build the HoloToolkit from the source, you'll need to clone the GitHub repository at https://github.com/Microsoft/HoloToolkit-Unity.git. If you've never cloned a repo before then you should consider using the GitHub desktop client, see https://desktop.github.com/.
+> Check the version of the MRTK asset for your target:
+> * 2017.1.2 for HoloLens
+> * 2017.2.0f3 for Immersive headsets
```

Comment:
I'm not sure if we want to add the version number here. It's just something else we have to remember to update.

Maybe copy the badges from the mainpage?

Technically both packages are exactly the same, with the only difference being the release notes and breaking changes depending on your need.

Reply:
nice idea.